caddytls/client.go

```diff
@@ -60,6 +61,11 @@ var newACMEClient = func(config *Config, allowPrompts bool) (*ACMEClient, error)
 		return nil, fmt.Errorf("%s: insecure CA URL (HTTPS required)", caURL)
 	}
 
+	// Set timeout by replacing ACME's http.Client
+	if HTTPTimeout > 1 {
+		acme.HTTPClient = http.Client{Timeout: time.Duration(HTTPTimeout) * time.Second}
```

Comment:
It's a package level variable and not connected to a single client instance. Assigning it once is enough.